packages/kbn-grid-layout/grid/use_grid_layout_events.ts

```diff
@@ -13,18 +13,42 @@ import { resolveGridRow } from './utils/resolve_grid_row';
 import { GridPanelData, GridLayoutStateManager } from './types';
 import { isGridDataEqual } from './utils/equality_checks';
 
+const MIN_SPEED = 50;
+const MAX_SPEED = 150;
+
 const scrollOnInterval = (direction: 'up' | 'down') => {
   let count = 0;
+  let currentSpeed = MIN_SPEED;
+  let maxSpeed = MIN_SPEED;
+  let turnAroundPoint: number | undefined;
+
   const interval = setInterval(() => {
-    // calculate the speed based on how long the interval has been going to create an ease effect
-    // via the parabola formula `y = a(x - h)^2 + k`
-    // - the starting speed is k = 50
-    // - the maximum speed is 250
-    // - the rate at which the speed increases is controlled by a = 0.75
-    const speed = Math.min(0.75 * count ** 2 + 50, 250);
-    window.scrollBy({ top: direction === 'down' ? speed : -speed, behavior: 'smooth' });
-    count++;
-  }, 100);
+    /**
+     * Since "smooth" scrolling on an interval is jittery on Chrome, we are manually creating
+     * an "ease" effect via the parabola formula `y = a(x - h)^2 + k`
+     *
+     * Scrolling slowly speeds up as the user drags, and it slows down again as they approach the
+     * top and/or bottom of the screen.
+     */
```

Comment:
The speed is calculated as shown in this diagram:

![image](https://github.com/user-attachments/assets/02b4389c-fe78-448d-9c02-c4ec5e722d5e)